package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "vscode-xlayers",
+  "name": "xlayers-vscode",
```

Comment:
We should put it under the xLayers organisation scope `@xlayers/vscode-extension`.

Reply:
I likt it 😄